services/cd-service/pkg/repository/repository.go

```diff
@@ -1659,7 +1659,9 @@ func (s *State) GetDeploymentMetaData(ctx context.Context, environment, applicat
 		if err != nil {
 			return "", time.Time{}, err
 		}
-		return result.Metadata.DeployedByEmail, result.Created, nil
+		if result != nil {
```

Comment:
What happens if the result is nil? What is returned in that case?

Reply:
fixed